pixy/httpapi.go

```diff
@@ -210,6 +289,16 @@ type consumeHTTPResponse struct {
 	Offset    int64  `json:"offset"`
 }
 
+type partitionOffsetView struct {
```

Comment:
Separation of concerns. The `PartitionOffset` in **admin.go** is not supposed to be concerned with serialization techniques used.